wiki/Welcome.md

```diff
@@ -0,0 +1,49 @@
+## Welcome do DragonProxy WIKI
+
+### 1. Presentation
+DragonProxy aim to be the first Minecraft PE <=> PC bridge, based on protocole translation.<br />
```

Comment:
aims* 
protocol*

to best the best* (you cant aim to be the first. you either are, or you arent. and we are.)

Minecraft: Bedrock Edition <=> Minecraft: Java Edition

Reply:
Thanks, my english is so bad sometimes ><